controllers/openstack_ansibleee_controller.go

```diff
@@ -293,10 +293,10 @@ func (r *OpenStackAnsibleEEReconciler) jobForOpenStackAnsibleEE(instance *redhat
 	}
 	if len(instance.Spec.Play) > 0 {
 		addPlay(instance, job)
-	} else if len(instance.Spec.Playbook) > 0 {
-		addPlaybook(instance, job)
 	} else if instance.Spec.Role != nil {
 		addRoles(instance, h, job)
+	} else if len(instance.Spec.Playbook) > 0 {
```

Comment:
can you add a comment here explaining why the order is imporant

Reply:
yes!